src/index.js

```diff
@@ -1 +1,43 @@
 import './style.css';
+
+const apiURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/xIniA5Uar0iFa7cLE5KD/scores/';
+
+const userInput = document.getElementById('user-input');
+const scoreInput = document.getElementById('score-input');
+const form = document.getElementById('form');
+const scoreList = document.getElementById('score-list');
+const refreshBtn = document.getElementById('refresh');
+
+const listElement = (user, number) => {
+  const li = document.createElement('li');
+  li.innerText = `${user} : ${number}`;
+  scoreList.appendChild(li);
+};
+
+form.addEventListener('submit', async (e) => {
+  e.preventDefault();
+  fetch(apiURL, {
+    method: 'POST',
+    body: JSON.stringify({
+      user: userInput.value,
+      score: scoreInput.value,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  }).then((response) => response.json());
+});
```

Comment:
- [ ] It looks like you forgot to use `await` in your function, I kindly invite you to add it. 🐱